From 74314e3a1466b2be65a290fe13eb6f161a0c4169 Mon Sep 17 00:00:00 2001
From: Kai Sommerfeld <kai.sommerfeld@gmx.com>
Date: Wed, 16 Oct 2019 09:08:34 +0200
Subject: [PATCH] Revert "[skin.estuary/pvr] Add support for more meta data to
 PVR info panel."

This reverts commit 14c1e50abd0bd550146cf4d8e9b7e4fbfe056af9.
---
 addons/skin.estuary/xml/Includes_PVR.xml | 74 ++++++++++++++----------
 1 file changed, 45 insertions(+), 29 deletions(-)

diff --git a/addons/skin.estuary/xml/Includes_PVR.xml b/addons/skin.estuary/xml/Includes_PVR.xml
index 79c1464f60e7..f5332fe331cd 100644
--- a/addons/skin.estuary/xml/Includes_PVR.xml
+++ b/addons/skin.estuary/xml/Includes_PVR.xml
@@ -191,30 +191,8 @@
 	<include name="PVRInfoPanel">
 		<control type="group">
 			<visible>!ListItem.IsFolder</visible>
-			<control type="label">
-				<top>120</top>
-				<width>830</width>
-				<height>262</height>
-				<font>font36_title</font>
-				<label>$INFO[ListItem.Title] $INFO[ListItem.Year,([COLOR grey],[/COLOR])]</label>
-				<visible>!PVR.HasEpg</visible>
-			</control>
-			<control type="label">
-				<top>120</top>
-				<width>830</width>
-				<height>262</height>
-				<font>font36_title</font>
-				<label>$INFO[ListItem.EpgEventTitle] $INFO[ListItem.Year,([COLOR grey],[/COLOR])]</label>
-				<visible>PVR.HasEpg</visible>
-			</control>
-			<control type="label">
-				<top>165</top>
-				<width>830</width>
-				<height>70</height>
-				<label>[I]$VAR[SeasonEpisodeLabel]$INFO[ListItem.EpisodeName][/I]</label>
-			</control>
 			<control type="image">
-				<top>250</top>
+				<top>135</top>
 				<left>630</left>
 				<width>200</width>
 				<height>200</height>
@@ -223,18 +201,24 @@
 				<fadetime>200</fadetime>
 			</control>
 			<control type="group">
-				<top>235</top>
+				<top>120</top>
 				<left>0</left>
 				<width>600</width>
 				<control type="label">
-					<height>150</height>
+					<height>262</height>
+					<font>font45</font>
+					<label>$INFO[ListItem.ChannelName]</label>
+				</control>
+				<control type="label">
+					<top>60</top>
+					<height>200</height>
 					<wrapmultiline>true</wrapmultiline>
-					<label>$INFO[ListItem.ChannelName,[B],[/B][CR]]$INFO[ListItem.Date,[COLOR grey]$LOCALIZE[552]:[/COLOR] ,[CR]]$INFO[ListItem.Duration,[COLOR grey]$LOCALIZE[180]:[/COLOR] ,[CR]]$INFO[ListItem.Genre,[COLOR grey]$LOCALIZE[515]:[/COLOR] ,[CR]]</label>
+					<label>$INFO[ListItem.TimerType,[COLOR grey]$LOCALIZE[803]:[/COLOR] ,[CR]]$INFO[ListItem.Date,[COLOR grey]$LOCALIZE[552]:[/COLOR] ,[CR]]$INFO[ListItem.Duration,[COLOR grey]$LOCALIZE[180]:[/COLOR] ,[CR]]$VAR[ExpirationDateTimeLabel]</label>
 				</control>
 				<control type="group">
 					<visible>Integer.IsGreater(ListItem.Progress,0)</visible>
-					<top>170</top>
-					<height>100</height>
+					<top>150</top>
+					<height>262</height>
 					<control type="label">
 						<label>$INFO[ListItem.StartTime]</label>
 					</control>
@@ -257,13 +241,45 @@
 					<visible>Integer.IsGreater(ListItem.PercentPlayed,0)</visible>
 				</control>
 			</control>
+			<control type="label">
+				<top>390</top>
+				<width>830</width>
+				<height>262</height>
+				<font>font36_title</font>
+				<label>$INFO[ListItem.Title] $INFO[ListItem.Year,([COLOR grey],[/COLOR])]</label>
+				<visible>!PVR.HasEpg</visible>
+			</control>
+			<control type="label">
+				<top>390</top>
+				<width>830</width>
+				<height>262</height>
+				<font>font36_title</font>
+				<label>$INFO[ListItem.EpgEventTitle] $INFO[ListItem.Year,([COLOR grey],[/COLOR])]</label>
+				<visible>PVR.HasEpg</visible>
+			</control>
+			<control type="label">
+				<top>435</top>
+				<width>830</width>
+				<height>70</height>
+				<label>[I]$VAR[SeasonEpisodeLabel]$INFO[ListItem.EpisodeName][/I]</label>
+			</control>
 			<control type="textbox">
 				<top>490</top>
 				<width>830</width>
 				<bottom>75</bottom>
-				<label>$INFO[ListItem.TimerType,[COLOR grey]$LOCALIZE[803]:[/COLOR] ,[CR]]$VAR[ExpirationDateTimeLabel]$VAR[PremieredLabel]$INFO[ListItem.Rating,[COLOR grey]$LOCALIZE[563]:[/COLOR] ,[CR]]$INFO[ListItem.Writer,[COLOR grey]$LOCALIZE[20417]:[/COLOR] ,[CR]]$INFO[ListItem.Director,[COLOR grey]$LOCALIZE[20339]:[/COLOR] ,[CR]]$INFO[ListItem.Cast,[COLOR grey]$LOCALIZE[206]:[/COLOR] ,[CR]][CR]$INFO[ListItem.Plot]</label>
+				<label>$INFO[ListItem.Plot]</label>
 				<autoscroll delay="10000" time="3000" repeat="10000">Skin.HasSetting(AutoScroll)</autoscroll>
 			</control>
+			<control type="textbox">
+				<top>530</top>
+				<width>830</width>
+				<height>340</height>
+				<align>center</align>
+				<font>font27</font>
+				<textcolor>80FFFFFF</textcolor>
+				<label>$LOCALIZE[19055]</label>
+				<visible>String.IsEmpty(Listitem.Plot) + String.IsEmpty(Listitem.Genre)</visible>
+			</control>
 		</control>
 		<control type="group">
 			<visible>ListItem.IsFolder</visible>
